### PR TITLE
Merge pull request #2 from WebDevManuel/dropdown

### PR DESCRIPTION
**Pull request by WebDevManuel**
**What i did:**
- Add a component to set the training volume without styles.

### CODE REVIEW

File: src/components/trainingInput/TrainingInput.module.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+label {
```
Comment (lajanzen) [inline src/components/trainingInput/TrainingInput.module.css:1]: In module CSS, you need to work with dot notations, so you also need classNames in your `TrainingInput.tsx`.

```suggestion
.label {
```

File: src/components/trainingInput/TrainingInput.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value;
+    setTraining({ ...training, [event.target.name]: value });
+    console.log(training);
+  };
+
+  return (
+    <form onSubmit={handleOnSubmit}>
+      <label htmlFor="sets">Sets</label>
```
Comment (lajanzen) [inline src/components/trainingInput/TrainingInput.tsx:35]: This goes for all 3 labels:

```suggestion
      <label className={styles.label} htmlFor="sets">Sets</label>
```

File: src/components/trainingInput/TrainingInput.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React, { ChangeEvent, FormEvent } from "react";
+import { useState } from "react";
+
```
Comment (lajanzen) [inline src/components/trainingInput/TrainingInput.tsx:3]:
```suggestion
import styles from "./TrainingInput.module.css";

```

File: src/components/trainingInput/TrainingInput.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+/* type DropdownProps = {
+  label: string;
+  idSelect: string;
+  options : OptionProps[];
+};*/
+
+/* type TrainingProbs = {
+  label: string;
+  value: number;
+}; */
```
Comment (lajanzen) [inline src/components/trainingInput/TrainingInput.tsx:13]: Do you still need this? 

```suggestion
```

**Comment by WebDevManuel** [comment]
Adoption of Lara's suggestions and add more styles

**Pull request merged**